src/elements/content-sidebar/activity-feed/activity-feed/ActiveState.js

```diff
@@ -62,27 +64,58 @@ const ActiveState = ({
                         </li>
                     );
                 case 'task':
-                    return item.task_assignment_collection.total_count ? (
-                        <li key={type + id} className="bcs-activity-feed-task">
-                            <Task
-                                {...item}
-                                currentUser={currentUser}
-                                getAvatarUrl={getAvatarUrl}
-                                getMentionWithQuery={getMentionWithQuery}
-                                getUserProfileUrl={getUserProfileUrl}
-                                mentionSelectorContacts={mentionSelectorContacts}
-                                onAssignmentUpdate={onTaskAssignmentUpdate}
-                                onDelete={onTaskDelete}
-                                onEdit={onTaskEdit}
-                                permissions={{
-                                    can_delete: true,
-                                    can_edit: true,
-                                }}
-                                // permissions are not part of task API so hard code to true
-                                translations={translations}
+                    return (
+                        <div>
```

Comment:
It looks like this is inside a `ul`, so using a div here will generate invalid markup.

Reply:
I'm pretty sure comments and old tasks use <p> and <div> internally, but I'll take a look to see if this is fixable for all

Reply:
They likely do contain other tags, but they're all inside of a parent `li` in this file. This change introduces a `div` wrapper around the `li` for tasks.

Reply:
Yes, but if they use `div` internally this is just as bad as here. I'll look into fixing it for all cases.

Reply:
I think there may be a misunderstanding. Below is the declaration for the Comment item type in this file, starting at line 49. Note that it has an `<li>` wrapper:

```
case 'comment':
    return (
        <li key={type + id} className="bcs-activity-feed-comment">
            <Comment
                {...item}
                currentUser={currentUser}
                getAvatarUrl={getAvatarUrl}
                getUserProfileUrl={getUserProfileUrl}
                onDelete={onCommentDelete}
                permissions={{
                    can_delete: getProp(permissions, 'can_delete', false),
                    can_edit: getProp(permissions, 'can_edit', false),
                }}
                translations={translations}
            />
        </li>
    );
```

Reply:
I see. I thought div was not allowed inside li.